7.7.поиск_ошибок_и_ревью_кода/7.7.4.py

```diff
@@ -0,0 +1,13 @@
+mx = 0
+s = 0
+for i in range(10):
```

Comment:
Вот тут уже `range()` верный используешь, молодец.

Однако так же лучше называть неиспользуемую переменную цикла как `_`.
```suggestion
for _ in range(10):
```